### DISCUSSION TITLE
youtube-geofind not working

### DISCUSSION THREAD

**Discussion by zyrex1010** [mattwright324/youtube-geofind#56] [General]
the website is not searching after submitting, anyone have same issue?

**Comment by mattwright324** [comment]
Hi @zyrex1010 

The api quota limit was reached. I can look into adding a custom message when that happens.

**Comment by zerolyj** [comment]
if you can make the webpage become an mobile  application,that's will cool

**Reply by mattwright324** [reply]
The site already works fine in a mobile web browser
